src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/CompilationErrorModel.cs

```diff
@@ -3,13 +3,14 @@
 
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.StackTrace.Sources;
 
-namespace Microsoft.AspNetCore.Diagnostics.Views
+namespace Microsoft.AspNetCore.Diagnostics.RazorViews
 {
     /// <summary>
     /// Holds data to be displayed on the compilation error page.
     /// </summary>
-    public class CompilationErrorPageModel
+    internal class CompilationErrorPageModel
```

Comment:
No, this is in a different namespace. The current types are present in Obsolete folder